test/functional/collection_tests.js

```diff
@@ -1581,6 +1581,44 @@ describe('Collection', function() {
     }
   });
 
+  it('should correctly perform estimatedDocumentCount on non-matching query', function(done) {
+    const configuration = this.configuration;
+    const client = configuration.newClient({ w: 1 }, { poolSize: 1, auto_reconnect: false });
```

Comment:
Can you import `MongoClient` directly, and use it instead of using `configuration.newClient`?